backend/src/apiserver/client/swf.go

```diff
@@ -89,5 +89,9 @@ func newOutOfClusterSwfClient() (*SwfClient, error) {
 
 	// create the clientset
 	swfClientSet, err := swfclient.NewForConfig(config)
+	if err != nil {
+		return nil, err
```

Comment:
can you add some prefix error message like "failed to create the client set"

Reply:
nit: correct me if I'm wrong, swfclient.NewForConfig's error message should already have similar information as "failed to create the client set", therefore I think that's unnecessary.

Reply:
This is the source code. https://github.com/kubeflow/pipelines/blob/b482ba83d8edf8e683f315bfcf3f700970b23129/backend/src/crd/pkg/client/clientset/versioned/clientset.go#L55. I think the error message inside may not be distinct when comparing to other error messages of creating client set

Reply:
I will wrap the error to make it more clear.